Added button to set a project as default

suprabathk opened the pull request:
Changes:
- Added a button to set a project as default.
- Added Indicator whether a project is default or not.

skks1212 reviewed (changes requested):
I am not able to distinguish between a disabled and active button. 
Secondary button with a border is not looking as nice.

suprabathk commented:
@skks1212 
I have made the necessary changes that you indicated:
- Removed hover effects and added disabled cursor for disabled buttons.
- Removed border for "Set as default" button
